packages/fcl/README.md

```diff
@@ -32,29 +32,24 @@ Having trouble with something? Reach out to us on [Discord](https://discord.gg/k
   - [Custodial Wallet Provider](./src/wallet-provider-spec/custodial.md) -- Documentation for Custodial Wallet Providers.
   - [Non-Custodial Wallet Provider](src/wallet-provider-spec/non-custodial.md) -- **Coming Soon** Documentation for Non-Custodial Wallet Providers.
 
-# Todo List
```

Comment:
Was removing this Todo List intentional?

Reply:
Yeah, it was feeling a little stable, and arguably has no value for the public.